packages/beacon-node/test/e2e/network/network.test.ts

```diff
@@ -247,7 +256,26 @@ describe("network", function () {
 
     expect(netA.gossip.getTopics().length).to.equal(0);
     netA.subscribeGossipCoreTopics();
-    expect(netA.gossip.getTopics().length).to.equal(5);
+    /**
+     * TODO DA confirm
+     * Topics from previous fork are still subscribed together
+     * with same topic of the new fork. Is this expected behaviour?
```

Comment:
sorry what is the confusion here, could you elaborate a bit more pls?

Reply:
for example after altair, you have both `'/eth2/f5a5fd42/beacon_block/ssz_snappy',` and `'/eth2/16abab34/beacon_block/ssz_snappy',` subscribed to. The comment was to confirm if this is fine, or if the old topics should be unsubscribed from, when the new topic with the new fork is subscribed to.